### ISSUE TITLE
Domain Replace With should always end with slash?

### ISSUE THREAD

**Issue by widoz** [inpsyde/search-and-replace#115]
Trying to make a replacement like:

Search For: `http://localhost/wp/search-and-replace/`
Replace With: `http://localhost/wp/search-and-replace`

I get correctly a new domain url but without slash the url become something like:
`http://localhost/wp/search-and-replacewp-content/...`

**Comment by bvdv** [comment]
If I understand correctly, S&R plugin should replace strictly "Search for:" to "Replace with:".  It is mean Domain Replace "With" should be filled strictly that user want to get, so slash is necessary. This question will be very useful for documentation as example for avoiding problems.

**Comment by bueltge** [comment]
Right, the function should do what the user add as input. It is in the responsibility of the user. Currently was the old idea to help them.
